modules/backend/formwidgets/Repeater.php

```diff
@@ -442,7 +442,19 @@ protected function getGroupFormFieldConfig($code)
             return null;
         }
 
-        $fields = array_get($this->groupDefinitions, $code.'.fields');
+        // Bring the config in to use all attributes in repeater
+        $group = $this->getConfig('groups', []);
+        if (is_string($group)) {
+            $group = $this->makeConfig($group);
+        }
+
+        if ($group && isset($group->{$code}['type']) && $group->{$code}['type'] === 'repeater') {
+            $fields = [
+                $code => $group->{$code}
```

Comment:
We are moving towards having trailing commas on all arrays in our code style now, to simplify Git diffs.
```suggestion
                $code => $group->{$code},
```